.github/workflows/release-create-hotfix.yml

```diff
@@ -15,17 +27,31 @@ jobs:
         with:
           app_id: ${{ secrets.GH_BOT_APP_ID }}
           private_key: ${{ secrets.GH_BOT_PRIVATE_KEY }}
+
+      - uses: actions/checkout@v4
```

Comment:
can you specify the fetch-depth if needed ?

Reply:
fetch-depth defaults to 1 which is fine for these purposes